src/app/pages/system/enclosure/components/views/enclosure-view/disks-overview/disks-overview.component.html

```diff
@@ -41,25 +41,44 @@
         </div>
         <div class="detail">
           <div class="label">{{ 'Model' | translate }}</div>
-          <div class="value">{{ slot().model }}</div>
+          <div class="value">{{ slot().model || ('N/A' | translate) }}</div>
         </div>
         <div class="detail">
           <div class="label">{{ 'Serial' | translate }}</div>
-          <div class="value">{{ slot().serial }}</div>
+          <div class="value">{{ slot().serial || ('N/A' | translate) }}</div>
         </div>
         <div class="detail">
           <div class="label">{{ 'Status' | translate }}</div>
-          <div class="value">{{ slot().pool_info?.disk_status }}</div>
+          <div class="value">{{ slot().pool_info?.disk_status || ('N/A' | translate) }}</div>
+        </div>
+        <div class="detail">
+          <div class="label">{{ 'Adv. Power Mgmt.' | translate }}</div>
+          <div class="value">{{ 'N/A' | translate }}</div>
+        </div>
+        <div class="detail">
+          <div class="label">{{ 'S.M.A.R.T. Enabled' | translate }}</div>
+          <div class="value">{{ 'N/A' | translate }}</div>
+        </div>
+        <div class="detail">
+          <div class="label">{{ 'S.M.A.R.T. Options' | translate }}</div>
+          <div class="value">{{ 'N/A' | translate }}</div>
```

Comment:
Why is this value hard coded? The response from `webui.enclosure.dashboard` is supposed to have all the disk details that we need for our enclosure.

Reply:
and if this is not something that is available in `webui.enclosure.dashboard`, it's okay to remove it from the UI.